src/lib/createLivePollMessage.ts

```diff
@@ -55,13 +55,14 @@ export async function createLivePollMessage(data: any, read: IRead, modify: IMod
             anonymousOptions,
         };
 
+        const timezone = data.user?.utcOffset | timeZone.value;
         const livePollEndTime = new Date();
         // Convert state.ttv to integer and add it to livePollEndTime
         livePollEndTime.setSeconds(livePollEndTime.getSeconds() + (+state.poll.ttv));
         poll.livePollEndTime = new Intl.DateTimeFormat(
             'en-GB',
             {
-                timeZone: timeZone.value,
+                timeZone: timezone.toString(),
```

Comment:
Are you sure this works? Considering, `utcOffset` is a numeric value, I don't see it being compatible here. Also tried it in a sandbox and it didn't work for me

![image](https://user-images.githubusercontent.com/34130764/224738450-8f4659a9-d996-4a0a-9e50-f78311e803ce.png)
ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat

Reply:
I am sorry about this actually I was not sure how can I test this.

Reply:
> I am sorry about this actually I was not sure how can I test this,

While it's okay for now, I would like to emphasise the importance of thoroughly testing any proposal before implementing it to avoid introducing new bugs. If you're not confident about the logic or the testing process, please do reach out and ask for help. It may take longer to implement something, but it's always better to ensure that it's done right than to rush and risk introducing new issues. Thank you for your attention to this matter.

Reply:
![image](https://user-images.githubusercontent.com/34130764/224907955-b2c19edb-b3a7-4043-ac06-87a3b398b928.png)

This solution seems to be working for an offset without decimal point. However it will likely fail for offset's like `5.5` (Indian UTC offset), `5.75` (Nepal's UTC offset). Please check these edge cases as well. 

PS: you can find a list of all timezones here -> https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

Reply:
@murtaza98  Sir, I tried to come up with a new approach to deal with this edge case I added simple logic if the timezone offset is not undefined and is not an integer, we calculate the time zone offset in minutes and add it to the UTC minutes of livePollEndTime using the setUTCMinutes() method. could you please take a look!